views/features.njk

```diff
@@ -0,0 +1,184 @@
+      <div class="usa-grid-full">
+        <div class="usa-width-one-half">
+          <img src="/images/illo-rocket-mountains.png" srcset="/images/illo-rocket-mountains@2x.png 2x">
+          <h3>Federalist</h3>
+
+          <div class="criteria-list">
+            <h5 class="criteria-list-heading">Federally compliant from the start</h5>
+            <p>Built and hosted on FedRAMP-certified Amazon Web Services (AWS) GovCloud infrastructure and cloud.gov.</p>
+            <p>Maintains a 3-year Low Impact Level Authority to Operate granted by GSA.</p>
+
+            <h5 class="criteria-list-heading">Reliable and secure</h5>
+            <p>Choose from different levels of administrator permissions, multiple levels of authentication, and protection against attacks.</p>
+            <p>Seamlessly support your custom URLs with Federalist’s content delivery network, keeping your site up and running always.</p>
+            <p>Drastically lower the probability and possibility of a security threat with static site hosting only.</p>
+
+            <h5 class="criteria-list-heading">Self-controlled content and code</h5>
+            <p>Deploy your content and code quickly and safely, all while under your control.</p>
+            <p>Move your sites on or off of Federalist at any time.</p>
+          </div>
+        </div>
+        <div class="usa-width-one-half">
+          <img src="/images/illo-pushing-stone.png" srcset="/images/illo-pushing-stone@2x.png 2x">
+          <h3>Other site publishers</h3>
+
+          <div class="criteria-list criteria-list--clock">
+            <h5 class="criteria-list-heading">Extensive security and compliance evaluations</h5>
+            <p>Demands an impact level assessment, system security plan, additional monitoring setup, vulnerability scanning, privacy threshold analysis, documentation review, penetration testing, and more to obtain an Authroity to Operate.</p>
+            <p>Requires an annual renewal if the system has a Moderate Impact.</p>
+
+            <h5 class="criteria-list-heading">Additional plugins required</h5>
+            <p>May require plug-ins to get the same benefits as Federalist, like a simple editorial and develop workflow or real-time publishing.</p>
+            <p>All plug-ins must go through their own security evaluation.</p>
+
+            <h5 class="criteria-list-heading">Less collaboration and low reusability</h5>
+            <p>Learn on your own and contribute less to the public without open source code or software.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
+  <div class="usa-grid">
+    <hr class="hr-light">
+    <section class="info-block">
+      <div class="usa-width-one-whole">
+        <h2>Customize and publish your site with confidence.</h2>
+        <p>With Federalist, you can:</p>
+        <ul class="checklist copy">
+          <li>Monitor your content creation process</li>
```

Comment:
Is it possible to position the icons to match the layout below for better visual alignment? There's a bit of an optical illusion happening that makes it seem like the text is drooping below the icon baseline. 

![screen shot 2018-07-23 at 6 08 19 pm](https://user-images.githubusercontent.com/11636908/43105604-7c576320-8ea3-11e8-9557-930f19204aa7.png)